src/diffusers/models/prior_transformer.py

```diff
@@ -92,18 +118,27 @@ def __init__(
             ]
         )
 
+        if self.config.out_dim is not None:
+            self.norm_in = nn.LayerNorm(inner_dim)
+        else:
+            self.norm_in = None
```

Comment:
```suggestion
        if norm_in_type == "layer":
            self.norm_in = nn.LayerNorm(inner_dim)
        elif norm_in_type is None:
            self.norm_in = None
       else:
           raise ValueError(f"{norm_in_type} does not exist.")
```

Reply:
Same here, let's create a new config type (it's not that horrible). We can't use config types that logically make no sense